account_manager/src/common.rs

```diff
@@ -37,3 +39,61 @@ pub fn base_wallet_dir(matches: &ArgMatches, arg: &'static str) -> Result<PathBu
         PathBuf::new().join(".lighthouse").join("wallets"),
     )
 }
+
+pub fn strip_off_newline_codes(bytes: &mut Vec<u8>) -> Vec<u8> {
```

Comment:
This function doesn't need to take a mutable reference to the `Vec` _and_ return a new `Vec`, it can just do one of the two, i.e.

```rust
// Option 1
fn strip_off_newlines(mut bytes: Vec<u8>) -> Vec<u8>;
// Option 2
fn strip_off_newlines(bytes: &mut Vec<u8>);
```

My preference is for option 1.

Reply:
I'd also like the function to be called `strip_off_newlines`, with a doc-comment describing (briefly) what it does, like:

```rust
/// Remove any number of newline or carriage returns from the end of a vector of bytes.
fn strip_off_newlines(..);
```